src/Frontend/Core/Engine/Footer.php

```diff
@@ -43,30 +43,33 @@ public function parse(): void
         $footerLinks = (array) Navigation::getFooterLinks();
         $this->template->assignGlobal('footerLinks', $footerLinks);
 
-        $siteHTMLFooter = (string) $this->get('fork.settings')->get('Core', 'site_html_footer', null);
-        $facebookAppId = $this->get('fork.settings')->get('Core', 'facebook_app_id', null);
+        $siteHTMLEndOfBody = (string) $this->get('fork.settings')->get('Core', 'site_html_end_of_body', $this->get('fork.settings')->get('Core', 'site_html_footer', null));
 
-        // facebook admins given?
+        // @deprecated remove this in Fork 6, Facebook should not be added automaticall
+       $facebookAppId = $this->get('fork.settings')->get('Core', 'facebook_app_id', null);
```

Comment:
The indentation is wrong here

Reply:
Fixed in 1383ce812dd095e9da37e47f53d0623538c2f1c5